products/bash/release-apic.sh

```diff
@@ -22,19 +22,21 @@
 #     ./release-apic.sh
 #
 #   Overriding the namespace and release-name
-#     ./release-apic.sh -n cp4i-prod -r prod
+#     ./release-apic.sh -n cp4i-prod -r prod -a false
 
 function usage() {
   echo "Usage: $0 -n <namespace> -r <release-name> [-t]"
 }
 
+tick="\xE2\x9C\x85"
+cross="\xE2\x9D\x8C"
 namespace="cp4i"
 release_name="ademo"
 tracing="false"
 production="false"
 CURRENT_DIR=$(dirname $0)
 
-while getopts "n:r:tp" opt; do
+while getopts "a:n:r:tp" opt; do
```

Comment:
Is `a:` needed here? I am assuming this is for HA_ENABLED but could not find that variable, may be missing?

Reply:
Nope. We originally did the HA stuff here, then I moved it to configure-apic-v10.sh, must have missed reverting this change. Now done!